mobile/src/main/java/org/openhab/habdroid/util/BaseActivity.java

```diff
@@ -0,0 +1,39 @@
+package org.openhab.habdroid.util;
```

Comment:
This one should live in the `ui` package.

Reply:
But isn't it just a utility?

Reply:
In a sense, yes. But it's also the base class to all activities, which clearly is UI. What I'd like to avoid is the util package becoming a dumping ground for stuff which 'doesn't have a better place', and as an activity (even an abstract one) clearly is UI, I think it should live there.